README.md

```diff
@@ -20,20 +20,23 @@ can be specified on the command line, or in the environment
 variable **RUCIO_REGISTER_CONFIG**.
 
 ```
-rucio-integration-register --repo /repo/main --collections HSC/runs/PDR2/PDR2-VVDS-FR/w_2024_02_DM-40654/step2a/group'*' --dataset-type visitSummary --rucio-dataset Dataset/HSC/runs/PDR2/PDR2-VVDS-FR/w_2024_02_DM-40654/step2a -C config.yaml
+rucio-integration-register --repo /repo/main/test --collections HSC/runs/PDR2/PDR2-VVDS-FR/w_2024_02_DM-40654/step2a/group'*' --dataset-type visitSummary --rucio-dataset Dataset/HSC/runs/PDR2/PDR2-VVDS-FR/w_2024_02_DM-40654/step2a -C config.yaml
```

Comment:
Better example would be to leave this as `/repo/main`, with scope `main` and `rse_root = /repo`.

Reply:
I made this change.  Also changed the scope from test to main in the config.yaml description several lines below this.